src/Microsoft.ML.TensorFlow/TensorFlowModel.cs

```diff
@@ -52,14 +52,14 @@ public DataViewSchema GetInputSchema()
         /// <summary>
         /// Scores a dataset using a pre-traiend <a href="https://www.tensorflow.org/">TensorFlow</a> model.
         /// </summary>
-        /// <param name="inputColumnName"> The name of the model input.</param>
-        /// <param name="outputColumnName">The name of the requested model output.</param>
+        /// <param name="inputColumnName"> The name of the model input. The data type is a vector of <see cref="System.Single"/>.</param>
```

Comment:
><see cref="System.Single"/> [](start = 101, length = 27)

Input can be of any  type. Not necessarily of float. Look at the following test
[https://github.com/dotnet/machinelearning/blob/f9f327ad314c2b175d9f676ef068a2ff1114c21d/test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs#L215](https://github.com/dotnet/machinelearning/blob/f9f327ad314c2b175d9f676ef068a2ff1114c21d/test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs#L215)